src/ota/ota_pal_test.c

```diff
@@ -342,7 +353,8 @@ TEST( Full_OTA_PAL, otaPal_Abort_FileWithBlockWritten )
 
     /* Signal that the download is being aborted. */
     xOtaStatus = otaPal_Abort( &xOtaFile );
-    TEST_ASSERT_EQUAL_INT( OtaPalSuccess, OTA_PAL_MAIN_ERR( xOtaStatus ) );
+    TEST_ASSERT( OTA_PAL_MAIN_ERR( xOtaStatus ) == OtaPalSuccess ||
```

Comment:
This test case tries to verify that aborting after writing a block to an open file. The implementation should return success. Otherwise, the next OTA job may fail when writing the the same file.

Reply:
Currently we can assume it is success.